etc/emqx.conf

```diff
@@ -2416,6 +2416,22 @@ broker.route_batch_clean = off
 ##  - false: disable trie path compaction
 # broker.perf.trie_compaction = true
 
+## Enable client disconnect event will be triggered by which reasons.
+## Value: on | off
+## `normal`: normal disconnected events(by client's disocnnect mqtt packet)
+## Default: on
+## `kickout`: client was kicked out manually with management API.
+## Default: on
+## `takeover`: session was takenover by another client with same client ID. (clean_session = false)
+## Default: off
+## `discard`: session was takeover by another client with same client ID. (clean_session = true)
+## Default: off
+##
+# broker.client_disconnect_normal = on
+# broker.client_disconnect_kicked = on
+# broker.client_disconnect_discarded = off
+# broker.client_disconnect_takeovered = off
```

Comment:
there is no compatibility issue, so we can STOP adding more typos, 
```suggestion
# broker.client_disconnect_takeover = off
```

Reply:
keep same style on 4.x
https://github.com/emqx/emqx/blob/v4.3.21/src/emqx_session.erl#L635-L640